src/share.js

```diff
@@ -0,0 +1,9 @@
+export const findItem = (array, id) => {
+    //const index = toolsArray.findIndex(tool => tool.id === id);
+    array.forEach(element => {
+        if(element.id === id) {
+            element.isFlipped = !element.isFlipped;
```

Comment:
Mutation of values should not have a place in modern JS code.

Please take a look at the concept of Pure Function. It is a very common concept in the JS world that may be asked of you to describe or implement one during a technical interview for a company. 

 https://blog.bitsrc.io/understanding-javascript-mutation-and-pure-functions-7231cc2180d3
https://medium.com/javascript-scene/master-the-javascript-interview-what-is-a-pure-function-d1c076bec976

This function could be written in multiple ways to make it non mutating on the passed array.  

Could you try to write 2 different implementation on how to do so?